dev-docs/source/GettingStarted.rst

```diff
@@ -50,12 +50,22 @@ Linux
 
 Red Hat/Cent OS/Fedora
 ~~~~~~~~~~~~~~~~~~~~~~
-Follow the `Red Hat instructions <http://download.mantidproject.org/redhat.html>`_ to add the
-stable release yum repository and then install the ``mantid-developer`` package:
+* Follow the `Red Hat instructions <http://download.mantidproject.org/redhat.html>`_ to add the
```

Comment:
If we're adding the `copr` repo below then we could remove this step as it only adds the ISIS repo that contains a subset of the packages anyway.